src/org/ods/component/ScanWithAquaStage.groovy

```diff
@@ -199,13 +215,28 @@ class ScanWithAquaStage extends Stage {
                 [ title: "Messages", value: prepareMessageToBitbucket(messages), ]
             ])
         }
+        if (actionableVulnerabilities?.size() > 0) {
+            if (data.messages) {
+                ((List) data.messages).add([
+                    title: "Blocking",
+                    value: "Yes"
+                ])
+            } else {
+                data.put("messages", [
```

Comment:
Try to avoid duplicate parts. If "messages" is needed, add it and later add the information